raiden/tests/integration/fixtures/blockchain.py

```diff
@@ -8,14 +8,14 @@
 from raiden.constants import GENESIS_BLOCK_NUMBER
 from raiden.network.proxies.proxy_manager import ProxyManager, ProxyManagerMetadata
 from raiden.network.rpc.client import JSONRPCClient
-from raiden.tests.utils.eth_node import (
+from raiden.tests.utils.network import jsonrpc_services
```

Comment:
Do we have a module called `tests.utils` and another module `utils.test_utils`? Should we rename or merge?

Reply:
the problem I need to solve here:
- `tests/utils` contains utilities for tests CI tests only and is stripped from shipped releases
- `smoketest` is broken in binary releases because...
- some of those utilities are shared between smoketest and `pytest`

therefore I moved those under the `raiden.utils` namespace, into the new `test_utils` module.

Reply:
@Dominik1999 how about calling the package `raiden.utils.test_support`?

Reply:
perfectamente